service/src/main/kotlin/io/provenance/digitalcurrency/consortium/stream/WebSocket.kt

```diff
@@ -35,7 +35,7 @@ data class Event(
 
 data class EventBatch(
     val height: Long,
-    val events: List<StreamEvent>
+    val events: Map<String, List<StreamEvent>>
```

Comment:
how did this change?

Reply:
batching transactions required me to handle a tx hash once and not once for each event. I'll make sure aggregation still works before I convert this to a real PR. Just needed a POC on a working system before implementing for omnibus usdf.

Reply:
I see. That makes sense. I thought this was the raw response from the websocket for some reason.

Reply:
i reverted this for the time being because i dont think it added much value, plus it removes the deterministic indexing that the list provides